<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Saurabhs-MacBook-Air.local" failures="0" tests="3" name="TestCaseExecution.TestCaseExecution" time="0.029" errors="1" timestamp="2025-04-02T16:11:43 IST" skipped="2">
  <testcase classname="TestCaseExecution.TestCaseExecution" name="beforeTest" time="0.029">
    <error message="/Users/saurabhmishra/eclipse-workspace/TrialArtifact\src\test\resources\config\locators.properties (No such file or directory)" type="java.io.FileNotFoundException">
      <![CDATA[java.io.FileNotFoundException: /Users/saurabhmishra/eclipse-workspace/TrialArtifact\src\test\resources\config\locators.properties (No such file or directory)
at java.base/java.io.FileInputStream.open0(Native Method)
at java.base/java.io.FileInputStream.open(FileInputStream.java:213)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:152)
at java.base/java.io.FileInputStream.<init>(FileInputStream.java:106)
at java.base/java.io.FileReader.<init>(FileReader.java:60)
at TestCaseExecution.TestCaseExecution.beforeTest(TestCaseExecution.java:36)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:657)
at org.testng.TestRunner.beforeRun(TestRunner.java:646)
at org.testng.TestRunner.run(TestRunner.java:612)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- beforeTest -->
  <testcase classname="TestCaseExecution.TestCaseExecution" name="TestCase_FilterPriceRange" time="0.000">
    <skipped/>
  </testcase> <!-- TestCase_FilterPriceRange -->
  <testcase classname="TestCaseExecution.TestCaseExecution" name="TestCase_SearchItem" time="0.000">
    <skipped/>
  </testcase> <!-- TestCase_SearchItem -->
</testsuite> <!-- TestCaseExecution.TestCaseExecution -->
